src/settings.ts

```diff
@@ -11,9 +11,22 @@ export enum AnimationSpeed {
   Fast,
 }
 
+export enum TrajectoryDiameter {
+  Fix_size,
+  Robot_size,
+}
+
+export enum TrajectoryColor {
+  Theme,
+  Robot_Color,
+  Shades,
+}
+
 export interface Settings {
   trajectoryAnimation: TrajectoryAnimation;
   trajectoryAnimationSpeed: AnimationSpeed;
+  trajectoryDiameter: TrajectoryDiameter;
+  trajectoryColor: TrajectoryColor;
```

Comment:
JFYI,

```
// Enums are not type safe =)
enum Num {
  zero = 0,
  one = 1,
  two = 2,
}
let num: Num = 3;
console.log(num)

```

Reply:
hmm interesting, not sure if this is a design decision or a bug in typescript. I think enum is appropriate for this use case though, the alternative would be to use string literals, but it makes passing in variables hard.

```ts
function foo(bar: 'bar'): void {}
const bar = 'bar';
foo(bar); // error, 'bar' is of type 'string', but expected type 'bar'
```